results/migrations/raw_data/2012_winners/data.json

```diff
@@ -0,0 +1,209 @@
+[
+    {
+        "film_title_en": "The Great Race",
```

Comment:
i think we need to calculate id to films based on lower case name + year
or id it to exist films data base with stub data

i will open separate ticket for that for future sprints